### PR TITLE
Make default elasticsearch client configurable.

### PR DESCRIPTION
**Pull request by blischalk**
In reference to issue #2.

### CODE REVIEW

File: elasticsearch/elasticsearch.rkt [hunk 1]
```diff
@@ -39,7 +39,11 @@
 (struct client (host port ssl? username password))
 
 ;;; Constants
-(define DEFAULT-CLIENT (client "localhost" 9200 #f #f #f))
+(define ES-USERNAME (if (getenv "ES_USERNAME") (getenv "ES_USERNAME") #f))
```
Comment (vishesh) [inline elasticsearch/elasticsearch.rkt]: You could just do `(define ES-USERNAME (getenv "ES_USERNAME"))` and for ES-PASSWORD.

**Comment by vishesh** [comment]
Line `(define ES-USERNAME (if (getenv "ES_USERNAME") (getenv "ES_USERNAME") #f))` should be replaced with `(define ES-USERNAME (getenv "ES_USERNAME"))` as getenv anyway returns a #f if entry is not found. Same for password. Rest looks good to me.

**Comment by blischalk** [comment]
Pull request updated with default false values from getenv.

**Comment by vishesh** [comment]
Thank you.

**Pull request merged**